syftlib/src/test/java/org/openmined/syft/unit/datasource/JobLocalDataSourceTest.kt

```diff
@@ -0,0 +1,136 @@
+package org.openmined.syft.unit.datasource
+
+import android.content.Context
+import android.content.res.AssetManager
+import com.google.protobuf.ByteString
+import com.nhaarman.mockitokotlin2.doReturn
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.verify
+import com.nhaarman.mockitokotlin2.whenever
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.spyk
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TemporaryFolder
+import org.openmined.syft.datasource.DIFF_SCRIPT_NAME
+import org.openmined.syft.datasource.JobLocalDataSource
+import org.openmined.syft.domain.SyftConfiguration
+import org.openmined.syftproto.execution.v1.PlanOuterClass
+import java.io.File
+import java.io.InputStream
+
+class JobLocalDataSourceTest {
+
+    @Rule
+    @JvmField
+    var tempFolder = TemporaryFolder()
+
+    @ExperimentalUnsignedTypes
+    @Test
+    fun `Given config getDiffScript returns correct asset stream`() {
+        val context = mockk<Context>()
+        val inputStream = mockk<InputStream>()
+        val assets = mockk<AssetManager>(){
+            every { open("torchscripts/$DIFF_SCRIPT_NAME") } returns inputStream
+        }
+        val config = mockk<SyftConfiguration>()
+        every { config.context } answers { context }
+        every { context.assets } answers { assets }
+        val cut = JobLocalDataSource()
```

Comment:
I like (it's a personal preference) to add a line before the `when` and the `then` blocks for readability.